packages/hierarchies-react/src/test/internal/TreeActions.test.ts

```diff
@@ -525,57 +579,61 @@ describe("TreeActions", () => {
 
       const actions = createActions(model);
 
-      actions.setInstanceFilter("root-1", filter);
-
-      expect(onModelChangedStub).to.be.calledOnce;
+      await actions.setInstanceFilter("root-1", filter)?.complete;
+      await waitFor(() => expect(onModelChangedStub).to.be.calledTwice);
```

Comment:
is this `waitFor` needed?

Reply:
removed waitFors that go after `?.complete`